pkgs/applications/misc/gnuradio/limesdr.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, boost, gnuradio
+, pythonSupport ? true, python, swig, limesuite
+} :
+
+assert pythonSupport -> python != null && swig != null;
+
+let
+  version = "1.0.0-RC";
+
+in stdenv.mkDerivation rec {
+  name = "gnuradio-limesdr-${version}";
+
+  src = fetchFromGitHub {
+    owner = "myriadrf";
+    repo = "gr-limesdr";
+    rev = "v${version}";
+    sha256 = "0b34mg9nfar2gcir98004ixrxmxi8p3p2hrvvi1razd869x2a0lf";
+  };
+
+  buildInputs = [
+    cmake boost gnuradio limesuite
+  ] ++ stdenv.lib.optionals pythonSupport [ python swig ];
```

Comment:
`cmake` definitely belongs in `nativeBuildInputs`. I'm not sure about `swig` - is it used at runtime or only at buildtime?

Reply:
Yes for `cmake`. That was a copy'n paste mistake (it's wrong in all other gr modules). With `swig` I am not a 100% sure but there are some derivations that put into `nativeBuildInputs`.